model/gen-ai/registry.yaml

```
@@ -177,6 +177,71 @@ groups:
               brief: 'Output tokens (completion, response, etc.)'
         brief: The type of token being counted.
         examples: ['input', 'output']
+      - id: gen_ai.agent.id   # alternatives: assistant (openai)
+        stability: experimental
```

nit, we're switching to `development`, mind changing things you add to it?

```suggestion
        stability: development
```

thanks!